v2/product/product-type.rst

```diff
@@ -1,4 +1,68 @@
 Create a product type
 =====================
+For this example we will create a product type for T-shirts
+## First create Product attribute
+First we will need to create a product attribute. We will create two product attributes - 
 
-@todo
+* Size
+* Color
+
+Goto ``admin/commerce/product-attributes``
+
+Click **Add product attribute**
+![](../images/product-attribute-1.png) 
+
+This will open a form.
+![](../images/product-attribute-2.png) 
+
+After saving this form, we are required to fill *attribute values*.
+![](../images/product-attribute-3.png) 
+
+Now save this form.
+
+Do the same for Color attribute.
+
+## Create Product variation
+Now we need a product variation for the image for t-shirts.
+
+Goto ``admin/commerce/config/product-variation-types``
```

Comment:
Again, it is helpful to help them find this in the menu system, help them click through this whole thing.